VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -171,4 +166,10 @@ protected void hideMessageInSitu(){
         }
     }
 
+    private void showErrorDialog(String message) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(message)
+                .setPositiveButton(R.string.label_ok, null)
+                .create().show();
```

Comment:
`AlertDialog.Builder` is a factory class, so might as well use it as such without creating the `builder` variable.